Kudu.Core/SiteExtensions/SiteExtensionManager.cs

```diff
@@ -758,8 +758,15 @@ private IEnumerable<SourceRepository> GetRemoteRepositories(string feedUrl)
             }
             else
             {
-                repos.Add(GetSourceRepository(DeploymentSettingsExtension.NuGetSiteExtensionFeedUrl));
-                repos.Add(GetSourceRepository(_settings.GetSiteExtensionRemoteUrl()));
+                // The remote feed URL can either be the default siteextensions.net, or some user override via
+                // SCM_SITEEXTENSIONS_FEED_URL. We only want to add nuget.org to the list if the user
+                // is *not* overriding the feed URL
+                string remoteUrl = _settings.GetSiteExtensionRemoteUrl(out bool isDefault);
```

Comment:
Does C# allow you to declare a variable as you pass it into a function, and it has scope outside the call to the function?  Wow...I've learned something new today!

Reply:
Yeah, it's a fairly recent addition. Very convenient!

Reply:
e.g. see https://www.codeproject.com/Tips/1175809/Csharp-New-Inline-Out-Variables